components/domains/steps/registerSteps.tsx

```
@@ -1,120 +1,136 @@
-          <DoneFilledIcon
-            width="16"
-            secondColor={theme.palette.primary.main}
-            color="#FFF"
-          />
-        )}
+        {steps.map((_, index) => (
+          <div
+            className={styles.stepsContainer}
+            role="status"
+            aria-label="Loading registration steps"
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)